spec/ama_layout/decorators/moneris_decorator_spec.rb

```diff
@@ -36,16 +36,19 @@
       expect(moneris_presenter.textbox).to include "color: rgba(0, 0, 0, 0.74902);"
       expect(moneris_presenter.textbox).to include "cursor: auto;"
       expect(moneris_presenter.textbox).to include "display: block;"
-      expect(moneris_presenter.textbox).to include "font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;font-size: 14px;"
-      expect(moneris_presenter.textbox).to include "font-style: normal;font-variant: normal;"
+      expect(moneris_presenter.textbox).to include "font-family: 'Open Sans', Helvetica, Roboto, Arial, sans-serif;"
+      expect(moneris_presenter.textbox).to include "font-size: 18px;"
+      expect(moneris_presenter.textbox).to include "font-style: normal;"
+      expect(moneris_presenter.textbox).to include "font-variant: normal;"
       expect(moneris_presenter.textbox).to include "font-weight: normal;"
-      expect(moneris_presenter.textbox).to include "height: 44px;"
+      expect(moneris_presenter.textbox).to include "height: 63px !important;"
```

Comment:
Is the !important necessary?

Reply:
Yes, there is a height being declared inline on all the fields, so until we can remove that height the important is necessary to override it.

Reply:
That sounds great!